viewer/packages/rendering/src/PointCloudMaterialManager.test.ts

```diff
@@ -52,4 +52,30 @@ describe('PointCloudMaterialManager', () => {
     expect(material1.blending).toBe(materialParameters.blending);
     expect(material2.blending).toBe(materialParameters.blending);
   });
+
+  test('material transparency can be toggled', () => {
+    const modelIdentifier = Symbol('model');
+    materialManager.addModelMaterial(modelIdentifier, objectData);
+
+    const material = materialManager.getModelMaterial(modelIdentifier);
+
+    material.transparent = true;
+    expect(material.transparent).toBe(true);
+
+    material.transparent = false;
+    expect(material.transparent).toBe(false);
```

Comment:
This is a strange test 😅 Why are we testing this? I don't see how it relates to the rest of the PR